### DISCUSSION TITLE
Path error when including relative expression in MoBi equation (related to Discussion #1151)

### DISCUSSION THREAD

**Discussion by NinaHanke** [Open-Systems-Pharmacology/Forum#1278] [Questions & Problems]
Dear community, 

We are modelling a compound that binds to VEGFA and undergoes TMDD. 
Maria included VEGFA expression (based on the gene expression database) in PK-Sim V11 and ticked only the “interstitial” box.

![Bild1](https://user-images.githubusercontent.com/36850897/184158538-c619e918-ed38-4449-8e71-e9b4680f2ade.png)

Then she exported the simulation to MoBi V11 and added a reaction for VEGFA turnover (called “turnover”). 
However, when putting interstitial relative expression in the turnover equation, we get the following error for all organs:

![Bild2](https://user-images.githubusercontent.com/36850897/184158719-f978eee5-ac04-4030-8960-085f0f24b3ca.png)

In the Parameter Start Values building block, relative expression in interstitial space is not listed. 
However, there are start values for intracellular relative expression (which should be zero):

![Bild3](https://user-images.githubusercontent.com/36850897/184158786-b3c2abea-459d-4e92-82ed-8947fce59386.png)

From the discussion https://github.com/Open-Systems-Pharmacology/Forum/discussions/1151#discussion-3898968 we understood  that there might be a way to adjust the PSVs - referring to msevestres comment:

![Bild4](https://user-images.githubusercontent.com/36850897/184158860-c57be120-f9ce-43fc-92ed-f99dadf6ece9.png)

However, we are not sure how to create a new entry for interstitial relative expression in our PSVs.

Please help! 
Maria and Nina

**Comment by Christoph27** [comment]
Dear Maria, Dear Nina,

it should not be necessary to extend your PSV (if you add in MoBi just a turnover reaction of a protein which was already defined in PK-Sim, all relative expressions should already be defined in the PSV after export from PK-Sim).
Reading https://github.com/Open-Systems-Pharmacology/Forum/discussions/1151#discussion-3898968 the NaNs still seem to be an issue ...
Since your protein is expressed only in the interstitial space, a possible workaround could be to add a condition at “Container Criteria” at the reaction, i.e. adding “tagged with Interstitial” at the turnover reaction.
Hope that helps ...


All the best, Christoph

**Comment by PavelBal** [comment]
Dear Nina,

one of the differences between the expression concepts in version 9 and 10+ is that all proteins are now present in all compartments. The relative expression parameter is always located in the intracellular space, and the resulting initial concentration is calculated from the relative expression value (which defines the expression in whole organ) and the fraction expressed in the specific compartment (in your case, fraction intracellular is 0 and fraction expressed in interstitial space is 1).

To model the turnover reaction, you can simply use the new parameter "Initial concentration" which already takes care of everything for you (including ontogeny factors etc).

Hope that helps :)
